README.md

```diff
@@ -6,8 +6,16 @@ whose members have write access to [Nixpkgs](https://github.com/nixos/nixpkgs).
 
 The [Nixpkgs commit delegators](https://github.com/orgs/NixOS/teams/commit-bit-delegation)
 maintain the member list in this repository.
-While it's in principle possible to request Nixpkgs commit permissions by creating a PR,
-please nominate yourself in [this issue](https://github.com/NixOS/nixpkgs/issues/321665) instead.
+
+## Nominations
+
+To nominate yourself or somebody else:
+1. Create an empty `members/<GITHUB_HANDLE>` file ([direct GitHub link](/../../new/main/members?filename=%3CGITHUB_HANDLE%3E))
+2. Create a PR with the change with the motivation in the PR description
```

Comment:
Should this also mention forking the repo?

I guess anyone nominating should be familiar enough with GitHub that they already understand forking.

Note: it was _these_ manual steps that I was proposing we could automate with a simple script; hopefully making it even simpler to nominate someone. Of course, that's tangential to establishing the workflow itself (this PR)